src/components/MerchantsPage/gam/CampaignListItem.tsx

```diff
@@ -1,68 +1,51 @@
-import ProgressBar from '../../ProgressBar';
+import CampaignProgressBar from './CampaignProgressBar';
 import styled from 'styled-components';
 import {
-    smallScreens,
-    tabletScreens,
+  smallScreens,
+  tabletScreens,
 } from '../../../utilities/general/responsive';
 import campaignDefaultImage from '../images/campaign_default.png';
 import apexLogo from '../images/apex-logo.png';
 import melonpannaLogo from '../images/melonpanna-logo.png';
 
 // In the final implementation, campaign will be object declared in types.ts
-const CampaignListItem = (campaign: String) => {
-    return (
-        <Container>
-            <ColumnContainer>
-                <img
-                    src={campaignDefaultImage}
-                    alt="campaign_image"
-                />
-            </ColumnContainer>
-            <ColumnContainer>
-                <Location>
-                    Sunset Park, Brooklyn
-                    </Location>
-                <Name>
-                    Melonpanna Tea & Shot x APEX for the Youth
-                    </Name>
-                <Description>
-                    Partnering with APEX for the Youth, we hope to raise 200 meals for underserved Asian and immigrant youth from low-income families.
-                    </Description>
-                <TimeStamp>
-                    Last contribution made 1h ago
-                    </TimeStamp>
-                <ProgressBar
-                    amountRaised={100}
-                    targetAmount={1000}
-                    progressBarColor={'#CF6E8A'}
-                    numContributions={20}
-                    numDonations={10}
-                    numGiftCards={10}
-                    donationAmount={50}
-                    giftCardAmount={50}
-                />
-            </ColumnContainer>
-            <ColumnContainer>
-                <ImagesContainer>
-                    <img
-                        src={apexLogo}
-                        alt="merchant_logo"
-                    />
-                    <img
-                        src={melonpannaLogo}
-                        alt="distributor_logo"
-                    />
-                </ImagesContainer>
-                <Button className='button--filled'>
-                    Visit merchant
-                    </Button>
-                <Button className={'button--outlined'}>
-                    Gift a meal
-                    </Button>
-            </ColumnContainer>
-        </Container>
-    );
+interface Props {
+  campaign: String;
+}
+
+const CampaignListItem = (campaign: Props) => {
+  return (
+    <Container>
+      <ColumnContainer>
+        <img src={campaignDefaultImage} alt="campaign_image" />
+      </ColumnContainer>
+      <ColumnContainer>
+        <Location>Sunset Park, Brooklyn</Location>
+        <Name>Melonpanna Tea & Shot x APEX for the Youth</Name>
+        <Description>
+          Partnering with APEX for the Youth, we hope to raise 200 meals for
```

Comment:
nit and probably unrelated as I think this is placeholder text? But isn't it "Apex for Youth", without the "the"?

Reply:
Ah yeah this placeholder. Will remove this once we hook this up with data from the backend.